examples/ibm-cloudant-database/modules/config-database/variables.tf

```diff
@@ -0,0 +1,23 @@
+
+
+variable "cloudant_instance_crn" {
+  description = "The cloudant instance CRN"
+  type        = string
+}
+
+variable "db_name" {
+  type        = string
+  description = "Database name"
+}
+
+variable "cloudant_database_partitioned" {
+  description = "Query parameter to specify whether to enable database partitions when creating a database."
+  type        = bool
+  default     = false
+}
+
+variable "cloudant_database_q" {
+  description = "The number of shards in the database. Each shard is a partition of the hash value range. Default is 8, unless overridden in the `cluster config`."
+  type        = number
+  default     = 8
```

Comment:
Wow do I wish this stuff wasn't so repetitive, some more `8` default shenanigans.